livelike/_attribution.py

```diff
@@ -0,0 +1,351 @@
+import re
+
+import numpy as np
+import pandas as pd
+
+
+def columns_to_labels(
+    df: pd.DataFrame,
+    filter: str,
```

Comment:
[Pre-commit is failing](https://results.pre-commit.ci/run/github/951924054/1746721171.Kee-9ujNQZCo2GDAWhIwCA) becauue `filter` argument shadows the Python [builtin `filter` function](https://docs.python.org/3/library/functions.html#filter)

```
livelike/_attribution.py:9:5: A002 Function argument `filter` is shadowing a Python builtin
   |
 7 | def columns_to_labels(
 8 |     df: pd.DataFrame,
 9 |     filter: str,
   |     ^^^^^^ A002
10 |     scrub: None | str = None,
11 |     keep_index: bool = True,
   |
```

It would be trivial to rename now before it goes into production. And make like more descriptive. Perhaps something like `filter_regex` or similar.

See also the comment below on `filter` as a variable.